mypyc/irbuild/classdef.py

```diff
@@ -205,7 +205,7 @@ def allocate_class(builder: IRBuilder, cdef: ClassDef) -> Value:
 MAGIC_TYPED_DICT_CLASSES = (
     'typing._TypedDict',
     'typing_extensions._TypedDict',
-)  # type: Final[Tuple[str, ...]]
```

Comment:
This changes the semantics, so we want to preserve the type. We want this to be an arbitrary-length tuple instead of a fixed-length tuple.